### PR TITLE
Finished adding query selectors

### PR DESCRIPTION
**Pull request by khan-osama**
https://user-images.githubusercontent.com/63980419/106502201-ea9ddf00-6478-11eb-9f69-82acd70d2fd0.mov

### CODE REVIEW

File: dom-querying/main.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+console.log('Hello, world!');
+
+document.querySelector('h1');
+
+console.log('Query for h1: ', document.querySelector('h1'));
```
Comment (codingcodymiller) [inline dom-querying/main.js]: This is duplicated.

File: dom-querying/main.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+console.dir(document.querySelector('h1'));
+
+document.querySelector('#explanation');
+
+console.log('Query for explanation: ', document.querySelector('#explanation'));
```
Comment (codingcodymiller) [inline dom-querying/main.js]: This is duplicated.

File: dom-querying/main.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+console.dir(document.querySelector('#explanation'));
+
+document.querySelector('.hint');
+
+console.log('Query for hint: ', document.querySelector('.hint'));
```
Comment (codingcodymiller) [inline dom-querying/main.js]: This is duplicated.

File: dom-querying/main.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+console.dir(document.querySelector('.hint'));
+
+document.querySelectorAll('p');
+
+console.log(document.querySelectorAll('p'));
```
Comment (codingcodymiller) [inline dom-querying/main.js]: This is duplicated.

File: dom-querying/main.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+document.querySelectorAll('.example-link');
+
+console.log(document.querySelectorAll('.example-link'));
```
Comment (codingcodymiller) [inline dom-querying/main.js]: This is duplicated.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Osama! You did a few steps multiple times, please remove the extras.

File: dom-querying/main.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+console.log('Hello, world!');
+
+document.querySelector('h1');
```
Comment (codingcodymiller) [inline dom-querying/main.js]: This should be logged using `console.log`

File: dom-querying/main.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+console.dir(document.querySelector('h1'));
+
+document.querySelector('#explanation');
```
Comment (codingcodymiller) [inline dom-querying/main.js]: This should be logged using `console.log`

File: dom-querying/main.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+console.dir(document.querySelector('#explanation'));
+
+document.querySelector('.hint');
```
Comment (codingcodymiller) [inline dom-querying/main.js]: This should be logged using `console.log`

File: dom-querying/main.js [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+
+console.dir(document.querySelector('.hint'));
+
+document.querySelectorAll('p');
```
Comment (codingcodymiller) [inline dom-querying/main.js]: This should be logged using `console.log`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by khan-osama** [comment]
Screen Shot 2021-02-01 at 11 06 50 AM

File: dom-querying/main.js (hunk 7 above)
Comment (codingcodymiller) [reply dom-querying/main.js]: Now that you've brought back the one I want, please remove this line :joy:

File: dom-querying/main.js (hunk 8 above)
Comment (codingcodymiller) [reply dom-querying/main.js]: Now that you've brought back the one I want, please remove this line :joy:

File: dom-querying/main.js (hunk 9 above)
Comment (codingcodymiller) [reply dom-querying/main.js]: Now that you've brought back the one I want, please remove this line :joy:

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**